src/View.php

```diff
@@ -586,7 +587,9 @@ public function beginPage(): void
         ob_start();
         PHP_VERSION_ID >= 80000 ? ob_implicit_flush(false) : ob_implicit_flush(0);
 
-        $this->eventDispatcher->dispatch(new PageBegin($this->getViewFile()));
+        if (is_string($viewFile = $this->getViewFile())) {
```

Comment:
How could it be non-string?

Reply:
> It could be false when rendering from View directly
> You can reproduce this behavior with the given method of the controller
> ` public function index(WebView $view, DataResponseFactoryInterface $responseFactory): ResponseInterface { return $responseFactory->createResponse($view->render('index')); }`

same